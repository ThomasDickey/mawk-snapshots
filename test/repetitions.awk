# $MawkId: repetitions.awk,v 1.3 2020/07/25 14:55:28 tom Exp $
# Test-script for MAWK
###############################################################################
# copyright 2016, jlp765
#
# This is a source file for mawk, an implementation of
# the AWK programming language.
#
# Mawk is distributed without warranty under the terms of
# the GNU General Public License, version 2, 1991.
###############################################################################
/b{,1}/ 
/ab{1,3}c/ 
/^ab{1,}c/ 
/ab{1,}c/ 
/ab{1}c/ 
/ab{2,3}c/ 
/a(b){2}c/ 
/a(b{2})c/ 
/ab{2}c/ 
/(ab){2,}c/ 
/ab{2,}c/ 
/ab{3}c/ 
/ab{3,5}c/ 
/(abb){1,}/ 
/b{1,}/ 
/b{2,}c/ 
/(b){2,}c/ 
/b{6,}c/ 
/b{7,}c/ 
/(bb){1,}c/ 
/(bb){1}c/ 
/bb{1}c/ 
/(bb){2,}c/ 
/(bb){2}c/ 
/bb{2}c/ 
/(bbb){1,}/ 
/(qwerty){4}/
/(qwerty){5}/
/(wabxcz){220}w/ 
/(wabxcz){220}/ 
/(wabxcz){221}/ 
/wabxcz{219}w/
