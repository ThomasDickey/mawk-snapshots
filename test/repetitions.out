abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
abc
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
ac
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
abbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbbbbbc
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
bbcab
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cabac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababac
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cababc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
cabc
xyz
xyz
xyz
xyz
xyz
xyz
xyz
xyz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
wabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxczwabxcz
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
qwertyqwertyqwertyqwerty
